react.js

```diff
@@ -51,6 +51,8 @@ module.exports = {
     'react/forbid-foreign-prop-types': hasPropTypes ? 'error' : 'off',
     'react/forbid-prop-types': 'off',
     'react/function-component-definition': 'off',
+    'react/hook-use-state': 'error',
```

Comment:
I don't care that much about this (mostly because sometimes it does make sense to be different and I don't want to have to disable it when that happens).

```suggestion
    'react/hook-use-state': 'off',
```